browser/ui/brave_browser_command_controller.cc

```diff
@@ -664,6 +666,15 @@ bool BraveBrowserCommandController::ExecuteBraveCommandWithDisposition(
     case IDC_BREAK_TILE:
       brave::BreakTiles(&*browser_);
       break;
+    case IDC_CLOSE_TAB:
+      {
+        int activeTabIndex = browser_->tab_strip_model()->active_index();
+        if( !browser_->tab_strip_model()->IsTabPinned(activeTabIndex) ) {
```

Comment:
We should filter this only when SharedPinnedTab feature flag is enabled

```suggestion
        if(base::FeatureList::IsEnabled(brave::tabs::kBraveSharedPinnedTabs) &&
          !browser_->tab_strip_model()->IsTabPinned(activeTabIndex) ) {
```

Reply:
got it!